tensorflow/core/kernels/scatter_op_test.cc

```diff
@@ -175,6 +186,47 @@ TEST_F(ScatterUpdateOpTest, Error_IndexOutOfRange) {
       << s;
 }
 
+TEST_F(ScatterSubOpTest, Error_IndexOutOfRange) {
+  MakeOp(DT_FLOAT_REF, DT_INT32);
+  // Feed and run
+  AddInputFromArray<float>(TensorShape({14}),
+                           {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0});
+  AddInputFromArray<int32>(TensorShape({3}), {0, 1, 99});
+  AddInputFromArray<float>(TensorShape({3}), {100, 101, 102});
+  Status s = RunOpKernel();
+  EXPECT_TRUE(
+      absl::StrContains(s.ToString(), "indices[2] = 99 is not in [0, 14)"))
+      << s;
+}
+
+TEST_F(ScatterSubOpTest, StressIndexTest) {
+  MakeOp(DT_INT32_REF, DT_INT32);
+  // Feed and run
+  const int kRows = 1;
+  std::vector<int32> values;
```

Comment:
For readability, we can initialize `std::vector` in the same line.
```suggestion
  std::vector<int32> values(kRows, 0);
```
Same for `indices` and `updates`.